### ISSUE TITLE
Only deprecated customVariablesWhitelist seems to work in WordPress.NamingConventions.ValidVariableName

### ISSUE THREAD

**Issue by mundschenk-at** [WordPress-Coding-Standards/WordPress-Coding-Standards#852]
Having this snippet in my `phpcs.xml`
```
<rule ref="WordPress.NamingConventions.ValidVariableName">
	<properties>
		<property name="customVariablesWhitelist" type="array" value="nodeValue" />
	</properties>
</rule>
```
correctly results in the warning `The customVariablesWhitelist property is deprecated in favor of customPropertiesWhitelist.` being thrown. However, when I replace the property, with `customPropertiesWhitelist`, it is simply ignored. I've added some error_log statements the actual sniff and it appears that the property is never set at all.

Did I miss something? Here's the snippet that fails to set the property:
```
<rule ref="WordPress.NamingConventions.ValidVariableName">
	<properties>
		<property name="customPropertiesWhitelist" type="array" value="nodeValue" />
	</properties>
</rule>
```

This is with `phpcs` 2.8.0 and the current `develop` branch of the sniffs.

**Comment by JDGrimes** [comment]
This should be fixed by #841 in the latest version of `develop` (just merged a few hours ago).

**Comment by mundschenk-at** [comment]
It is.
